independent-publisher-connectors/openFDA Drug/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1104 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "openFDA Drug",
```

Comment:
@last72 Can it be just "openFDA" or "OpenFDA" ?

Reply:
Hi @sriyen-msft the openFDA API includes many endpoints and it can be categorised to:
- animalandveterinary
- drug
- device
- food
- other
- tobaco

There are too many actions and endpoints if I include all of them so started the drug one first. It would be appropriate to append "Drug" at the end of it.

Reply:
@last72 Thank you so much for the clarification.